packages/client/components/Insights.tsx

```diff
@@ -44,6 +46,12 @@ const Insights = (props: Props) => {
   )
   const {viewer} = data
   const {domains} = viewer
+  const atmosphere = useAtmosphere()
+  domains.forEach(({id: domainId}) => {
```

Comment:
I got a bunch of bad merge conflicts in https://github.com/ParabolInc/parabol/pull/6764 after https://github.com/ParabolInc/parabol/issues/6681 was merged to master, so I closed that PR and created a fresh one as there aren't too many changes.

In your previous review ([see comment](https://github.com/ParabolInc/parabol/pull/6764/commits/9d2fc6134929630cee81d43180d0c5acde5e3b7c#r899226658)), you said that we should use the `domainId` here instead of `companyId`, so I've pushed this update here.